GenderPayGap.WebUI/Classes/BaseClasses/BaseController.cs

```diff
@@ -572,7 +572,8 @@ protected IActionResult CheckUserRegisteredOk(out User currentUser)
                 return RedirectToAction(nameof(OrganisationController.ManageOrganisations), "Organisation");
             }
 
-            if (userOrg.Organisation.SectorType == SectorTypes.Private)
+            if (userOrg.Organisation.SectorType == SectorTypes.Private 
+                && !FeatureFlagHelper.IsFeatureEnabled(FeatureFlag.PrivateManualRegistration))

```

Comment:
I'm a bit concerned about the placement of this.

This method is called from lots of actions throughout the site, to check if the user is allowed to perform that action.
At the end of this method, there's a `return null` which means "Good to continue with this action".
If we want to prevent the user doing an action (e.g. submitting data for an organisation before their registration is confirmed) then we need to return a `RedirectToAction` somewhere in this method.

I think the change you've made might allow the user to perform actions when they hadn't yet been verified.
e.g. if we have this feature flag turned on, then we omit all the checks between lines 577 and 621.

I think, instead, we should add the feature flag just around line 587 - I'm not sure where we should redirect them to, perhaps the "we have your details and will be in touch" page.